code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -50,7 +48,15 @@
 			/mob/living/simple_animal/hostile/blob,
 			/mob/living/simple_animal/ascendant_shadowling
 			)//exclusion list for things you don't want the reaction to create.
-		var/list/critters = typesof(/mob/living/simple_animal/hostile) - blocked // list of possible hostile mobs
+		var/list/meancritters = typesof(/mob/living/simple_animal/hostile) - blocked // list of possible hostile mobs
+		var/list/nicecritters = list(typesof(/mob/living/simple_animal/pet),

```

Comment:
You don't have to add the pets manually.
you just have to make the ```typesof(/blah/blah/pet)``` be ADDED to the list, instead of as an element of the list.